### PR TITLE
Circle Analytics

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

**Review by devformatters — APPROVED** [review]
Go. 💆🏼‍♂️

File: src/types/circle.ts [hunk 1]
```diff
@@ -81,16 +77,16 @@ export default /* GraphQL */ `
...
     "Invitations belonged to this Circle."
-    invites: Invites!
+    invites: Invites! @auth(mode: "${AUTH_MODE.oauth}")
 
     "Invitation used by current viewer."
     invitedBy: Invitation
 
     "Analytics dashboard."
-    analytics: CircleAnalytics!
+    analytics: CircleAnalytics! @auth(mode: "${AUTH_MODE.oauth}")
```
Comment (devformatters) [inline src/types/circle.ts:89]: I thought `@auth` can only be applied under `viewer`  🤦🏼

Comment (robertu7) [reply src/types/circle.ts]: You are right, will fix it.

**Review by PhilHua — APPROVED** [review]

**Review by jeremyOk — APPROVED** [review]

File: src/types/circle.ts (hunk 1 above)
Comment (robertu7) [reply src/types/circle.ts]: fixed

**Pull request merged**